apps/dotcom/scripts/build.ts

```diff
@@ -56,6 +56,32 @@ async function build() {
 	await exec('cp', ['-r', 'dist', '.vercel/output/static'])
 	await exec('rm', ['-rf', ...glob.sync('.vercel/output/static/**/*.js.map')])
 
+	// Add fonts to preload into index.html
+	const assetsList = (await exec('ls', ['-1', 'dist/assets'])).split('\n').filter(Boolean)
+	const fontsToPreload = [
```

Comment:
Hm, good point - feels like it would be tricky to have these two scripts hand-off that info though. Do you have a suggestion on how to do that cleanly? 🤔 Feels a little tricky.

Reply:
Yeah, feels tricky. Tbh it feels like tldraw package should expose this somehow? Still wouldn't be completely straight forward to get the info in the scripts though.